### ISSUE TITLE
Serveral test failures on current codebase

### ISSUE THREAD

**Issue by haukex** [cpants/www-cpants#112]
I just wanted to report a few test failures that I'm getting:
```
t/kwalitee/has_separate_license_file.t ................. 4/? 
#   Failed test 'IKEGAMI/Syntax-Feature-Void-v1.0.0.tar.gz has_separate_license_file: 0'
#   at .../lib/WWW/CPANTS/Test.pm line 92.
#          got: '0'
#     expected: '1'
# Looks like you failed 1 test of 11.

t/kwalitee/no_pod_errors.t ............................. 12/? 
#   Failed test 'DCANTRELL/Number-Phone-3.3000.tar.gz no_pod_errors: 0'
#   at .../lib/WWW/CPANTS/Test.pm line 92.
#          got: '0'
#     expected: '1'
# Looks like you failed 1 test of 13.

t/kwalitee/use_strict.t ................................ 14/? 
#   Failed test 'NINE/Inline-Perl6-0.07.tar.gz use_strict: 0'
#   at .../lib/WWW/CPANTS/Test.pm line 92.
#          got: '0'
#     expected: '1'
# Looks like you failed 1 test of 15.

t/kwalitee/use_warnings.t .............................. 12/? 
#   Failed test 'NINE/Inline-Perl6-0.07.tar.gz use_warnings: 0'
#   at .../lib/WWW/CPANTS/Test.pm line 92.
#          got: '0'
#     expected: '1'
# Looks like you failed 1 test of 13.

t/kwalitee/valid_signature.t ........................... 12/? 
#   Failed test at t/kwalitee/valid_signature.t line 22.
# Looks like you failed 1 test of 13.
```

**Comment by charsbar** [comment]
Thanks for the report. Fixed valid_signature breakage. As for use_strict and use_warnings, because I'm replacing Module::ExtractUse with Perl::PrereqScanner::NotQuiteLite behind the scene, I'll leave them until I finish it. The rest seem to be working under my environment (because of several compatibility reasons, I'm still running CPANTS using Perl 5.20, but both passed with Perl 5.28 and 5.26). Could you let me know the versions of Archive::Tar and Pod::Simple?

**Comment by haukex** [comment]
```
$ perl -MArchive::Tar -le 'print $Archive::Tar::VERSION'
2.24
$ perl -MPod::Simple -le 'print $Pod::Simple::VERSION'
3.35
```

I upgraded `Archvie::Tar` to 2.32 and fetched the latest version of the repo (44ee9164fede1520f4c51442656409d3be83e6ed). The `valid_signature.t` failure seems to be fixed, the other four test failures are still happening, same output as above (Perl 5.26 on Linux).

I just ran `Module::CPANTS::Analyse` 0.96 on the downloaded `IKEGAMI/Syntax-Feature-Void-v1.0.0.tar.gz`, and am also getting `has_separate_license_file => 0` in the output, with `license => "unrestricted defined in META.yml"`. Could the issue be with `Module::CPANTS::Kwalitee::License`?

When I run `t/kwalitee/no_pod_errors.t`, I see as part of the output: `{ 'error' => { 'no_pod_errors' => ".../Number-Phone-3.3000/lib/Number/Phone/UK/Data.pm --  Around line 1125:  Non-ASCII character seen before =encoding in ...`. I don't see any mention of POD errors in the `Module::CPANTS::Analyse` output, I'm not sure where to go from there.

**Comment by charsbar** [comment]
Aha. All the remaining issues were because you used Module::CPANTS::Analyse 0.96.

Relevant commits are:
https://github.com/cpants/Module-CPANTS-Analyse/commit/9c07c438423590cb6ae4dcb4365929e5dbf47713 for no_pod_errors, https://github.com/cpants/Module-CPANTS-Analyse/commit/7c81f04a12699db823e73e572f188f75c64b483b for both use_strict and use_warnings and https://github.com/cpants/Module-CPANTS-Analyse/commit/6018f9778c42576041fa46bea35d3a9e19cb3467 for has_separate_license_file, and I seemed to have forgotten to mention the first in Changes when I released 0.97_04.

I'm hoping I can ship the stable version around the end of this month.
